oblib/taxonomy.py

```diff
@@ -91,6 +103,25 @@ def __repr__(self):
             "}"
 
 
+class Relationship(object):
+    """Relationship holds a taxonomy relationship record."""
+
+    def __init__(self):
+        """Relationship constructor."""
+        self.role = None
+        self.from_ = None
```

Comment:
Is there a reason for `from_` rather than `from`?  I don't think it will shadow the reserved `from`

Reply:
The IDE highlighted "from" so I assumed that it was improper and changed it to "from_".  I can move it back if you prefer.  Thoughts?

Reply:
Its not the best name for an attribute but it won't conflict with `from`.  I think it's awkward to have `.from_` and `.to`

Reply:
I'd change to `.from` and not worry about PyCharm's flagging it.